akka-stream/src/main/scala/akka/stream/impl/io/compression/DeflateDecompressorBase.scala

```diff
@@ -10,14 +10,14 @@ import akka.stream.impl.io.ByteStringParser.{ ParseResult, ParseStep }
 import akka.util.ByteString
 
 /** INTERNAL API */
-private[akka] abstract class DeflateDecompressorBase(maxBytesPerChunk: Int = DeflateDecompressorBase.MaxBytesPerChunkDefault)
+private[akka] abstract class DeflateDecompressorBase(maxBytesPerChunk: Int)
   extends ByteStringParser[ByteString] {
 
   abstract class DecompressorParsingLogic extends ParsingLogic {
     val inflater: Inflater
     def afterInflate: ParseStep[ByteString]
     def afterBytesRead(buffer: Array[Byte], offset: Int, length: Int): Unit
-    val inflateState: Inflate
+    val Inflating: Inflate
```

Comment:
I changed it... hope you don't mind @jrudolph - `abstract val` also are a bit scary, made it a def